shellscript/bash-linux/s3/bucket-lifecycle-operations/README.rst

```diff
@@ -0,0 +1,52 @@
+.. Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0
+   International License (the "License"). You may not use this file except in compliance with the
+   License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+   either express or implied. See the License for the specific language governing permissions and
+   limitations under the License.
+
+#####################################
+Amazon S3 bucket lifecycle operations
+#####################################
+
+This example demonstrates how to interact with some of the basic Amazon S3 operations. It
```

Comment:
"interact with some of the basic" -> "perform some"

Reply:
Fixed.